services/common/src/interfaces/mineDocument.interface.ts

```diff
@@ -0,0 +1,11 @@
+export interface IMineDocument {
```

Comment:
@matbusby-fw @henryoforeh-aot @isuru-aot @taraepp @simensma-fresh  I was looking to setup the interface for MineDocument on my end as it is part of the response body of one of the functions in permit's actionCreator. I notice in the core-api response_models.py file the MINE_DOCUMENT_MODEL has a structure like this:

```
MINE_DOCUMENT_MODEL = api.model(
    'MineDocument', {
        'mine_document_guid': fields.String,
        'mine_guid': fields.String,
        'document_manager_guid': fields.String,
        'document_name': fields.String,
        'upload_date': fields.DateTime,
        'create_user': fields.String
    })
```

I am wondering now is there a difference in how response types are structured versus a type with the same name used in the reducer file? If so should we setup a folder called something like responses in the interface directory and place all the response types in there? or maybe I am over thinking this, please let me know.

Reply:
Ah I see, ok thanks for clarifying I thought maybe there were two types with the same name in use but seperated.